test/e2e/serial/tests/workload_placement_nodelabel.go

```diff
@@ -0,0 +1,416 @@
+				deployment.Spec.Template.Spec.Affinity = affinity
+				klog.Infof("create the test deployment with requests %s", e2ereslist.ToString(requiredRes))
+				err = fxt.Client.Create(context.TODO(), deployment)
+				Expect(err).NotTo(HaveOccurred(), "unable to create deployment %q", deployment.Name)
+
+				By("waiting for deployment to be up & running")
+				dpRunningTimeout := 1 * time.Minute
+				dpRunningPollInterval := 10 * time.Second
+				err = e2ewait.ForDeploymentComplete(fxt.Client, deployment, dpRunningPollInterval, dpRunningTimeout)
+				Expect(err).NotTo(HaveOccurred(), "Deployment %q not up & running after %v", deployment.Name, dpRunningTimeout)
+
+				By(fmt.Sprintf("checking deployment pods have been scheduled with the topology aware scheduler %q and in the proper node %q", serialconfig.Config.SchedulerName, targetNodeName))
+				pods, err := schedutils.ListPodsByDeployment(fxt.Client, *deployment)
+				Expect(err).NotTo(HaveOccurred(), "Unable to get pods from Deployment %q: %v", deployment.Name, err)
+				for _, pod := range pods {
+					Expect(pod.Spec.NodeName).To(Equal(targetNodeName), "pod %s/%s is scheduled on node %q but expected to be on the target node %q", pod.Namespace, pod.Name, targetNodeName)
+					schedOK, err := nrosched.CheckPODWasScheduledWith(fxt.K8sClient, pod.Namespace, pod.Name, serialconfig.Config.SchedulerName)
+					Expect(err).ToNot(HaveOccurred())
+					Expect(schedOK).To(BeTrue(), "pod %s/%s not scheduled with expected scheduler %s", pod.Namespace, pod.Name, serialconfig.Config.SchedulerName)
+				}
+
+				By("Verifing the NRT statistics are updated")
+				targetNodeNRTCurrent, err := e2enrt.FindFromList(nrtCandidates, targetNodeName)
+				Expect(err).NotTo(HaveOccurred())
+				Expect(e2enrt.CheckEqualAvailableResources(*targetNodeNRTInitial, *targetNodeNRTCurrent)).To(BeTrue())
+
+				By("unlabel nodes during execution and check that the test's pod was not evicted due to shaked matching creteria")
+				err = unlabelTarget()
+				if err != nil {
+					klog.Errorf("Error while trying to unlabel node %q. %v", targetNodeName, err)
+				}
+				err = unlabelAlternative()
+				if err != nil {
+					klog.Errorf("Error while trying to unlabel node %q. %v", alternativeNodeName, err)
+				}
+
+				//check that it didn't stop running for some time
+				By(fmt.Sprintf("ensuring the deployment %q keep being ready", deployment.Name))
+				Eventually(func() bool {
+					updatedDp := &appsv1.Deployment{}
+					err = fxt.Client.Get(context.TODO(), client.ObjectKeyFromObject(deployment), updatedDp)
+					Expect(err).ToNot(HaveOccurred())
+					return e2ewait.IsDeploymentComplete(deployment, &updatedDp.Status)
+				}, time.Second*30, time.Second*5).Should(BeTrue(), "deployment %q became unready", deployment.Name)
+
+			})
+
+			It("[test_id:49843][tier2] should be able to schedule pod with affinity property prefferdDuringSchedulingIgnoredDuringExecution on the available node with feasible numa zone", func() {
```

Comment:
this test case and 47597 can be part of Describe Table since both are performing same checked but different node affinity configurations.

Reply:
they can. This will probably require some test code refactoring, though.